protocols/data_processor.py

```
@@ -4,101 +4,105 @@
                         if isinstance(data, list):
                             total_records += len(data)
                         elif isinstance(data, dict):
                             total_records += 1
                         processed_count += 1
-                        insights.append(f"{filename}: {type(data).__name__} with {len(data) if isinstance(data, (list, dict)) else 1} items")
-                except:
+                        insights.append(
+                            f"{filename}: {
+                                type(data).__name__} with {
+                                len(data) if isinstance(
+                                    data, (list, dict)) else 1} items"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)